lib/rev-api/models/order_request.rb

```diff
@@ -152,6 +152,9 @@ class CaptionOptions < InputOptions
     # Array of file formats the captions should be delivered as.  (Optional, default is SubRip)
     attr_reader :output_file_formats
 
+    # Optional, a list of comma-separated language codes to request foreign language subtitles
+    attr_reader :subtitle_languages
```

Comment:
As discussed over sh, let's switch the server api to accept a list of languages, not a comma-separated string.